packages/x-components/src/x-modules/identifier-results/components/identifier-result.vue

```diff
@@ -57,14 +57,14 @@
      * @public
      */
     protected get highlightedQueryHTML(): string {
-      const identifierValue = this.result.identifier.value;
+      const identifierValue = this.result.identifier?.value;
       if (identifierValue && this.identifierHighlightRegexp) {
         return identifierValue.replace(
           this.identifierHighlightRegexp,
           '<span class="x-identifier-result__matching-part">$1</span>'
         );
       }
-      return identifierValue;
+      return identifierValue ?? '';
```

Comment:
If you end up moving the nullish coalescing operator to the first line:

```suggestion
      return identifierValue;
```